modules/ROOT/pages/clustering/settings.adoc

```diff
@@ -45,62 +45,28 @@ Possible values are:
 
 [.compact]
 `LIST`::
-Treats `dbms.cluster.discovery.endpoints` (or `dbms.cluster.discovery.v2.endpoints`, if you use discovery service v2 available as of Neo4j 5.23) as a list of addresses of servers to contact for discovery.
+Treats `dbms.cluster.endpoints` as a list of addresses of servers to contact for discovery.
 `DNS`::
-Treats `dbms.cluster.discovery.endpoints` (or `dbms.cluster.discovery.v2.endpoints`, if you use discovery service v2 available as of Neo4j 5.23) as a domain name to resolve via DNS.
-Expect DNS resolution to provide A records with hostnames or IP addresses of servers to contact for discovery, on the port specified by `dbms.cluster.discovery.endpoints`.
+Treats `dbms.cluster.endpoints` as a domain name to resolve via DNS.
+Expect DNS resolution to provide A records with hostnames or IP addresses of servers to contact for discovery, on the port specified by `dbms.cluster.endpoints`.
 `SRV`::
-Treats `dbms.cluster.discovery.endpoints` (or `dbms.cluster.discovery.v2.endpoints`, if you use discovery service v2 available as of Neo4j 5.23) as a domain name to resolve via DNS.
+Treats `dbms.cluster.endpoints` as a domain name to resolve via DNS.
 Expect DNS resolution to provide SRV records with hostnames or IP addresses and ports, of servers to contact for discovery.
 `K8S`::
 Accesses the Kubernetes list service API to derive addresses of servers to contact for discovery.
 Requires `dbms.kubernetes.label_selector` to be a Kubernetes label selector for Kubernetes services running a server each and `dbms.kubernetes.service_port_name` to be a service port name identifying the discovery port of cluster servers services.
```

Comment:
Need to add `discovery` in to the setting name, this is the V1 name.
```suggestion
Requires `dbms.kubernetes.label_selector` to be a Kubernetes label selector for Kubernetes services running a server each and `dbms.kubernetes.discovery.service_port_name` to be a service port name identifying the discovery port of cluster servers services.
```

Also need to remove `dbms.kubernetes.service_port_name` from this settings page.